projects/Mallard/src/components/article/index.tsx

```diff
@@ -37,6 +38,25 @@ const styles = StyleSheet.create({
     },
 })
 
+const render = article => {
+    return `
+    <html>
+    <head></head>
+    <body>
+      ${article
+          .filter(el => el.type === 0)
+          .map(el => el.textTypeData.html)
+          .join('')}
+      <script>
+        setTimeout(function() {
```

Comment:
You might wanna use requestanimationframe or window.onload to get a more accurate measure;

Reply:
Ha, yeah good idea!